emails/models.py

```diff
@@ -363,3 +372,17 @@ def delete(self, *args, **kwargs):
         self.user_profile.num_address_deleted += 1
         self.user_profile.save()
         return super(DomainAddress, self).delete(*args, **kwargs)
+
+    @property
+    def full_address(self):
+        return '%s@%s.%s' % (
+            self.address, self.user_profile.subdomain, DEFAULT_DOMAIN
+        )
+
+
+class Reply(models.Model):
+    relay_address = models.ForeignKey(RelayAddress, on_delete=models.CASCADE, blank=True, null=True)
+    domain_address = models.ForeignKey(DomainAddress, on_delete=models.CASCADE, blank=True, null=True)
```

Comment:
Not a big problem if we do not have the reply object be editable by API endpoints.